fedora_messaging/message.py

```diff
@@ -510,3 +618,18 @@ def flatpaks(self):
             list(str): A list of affected flatpaks names.
         """
         return []
+
+    def dump(self):
```

Comment:
I wonder if this is going to be an API most people want to use, or if they should only use the "dumps" and "loads" interfaces outside the class. Maybe we should mark this as private (call it ``_dump``) initially so we don't pollute the namespace with stuff most people won't want to use. What do you think?

Reply:
As far as I know(c++ exp) private methods are not available outside the class(except friends). So, if we are using "Message.dump" in "dumps" function, it means that we have to allow access from outside. I don't have a big experience in python and I know that underscore(even dunder) can't prevent clients from access private methods, so I'm not sure about final solution. I would like to let you decide in this topic.

Reply:
Ah, yes, Python has no way to force clients to not touch internals of objects, it's more like guidelines. I think we should make it private (to begin with, anyway) since I can't think of a reason people should be using it.